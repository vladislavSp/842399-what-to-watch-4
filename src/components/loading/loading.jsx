import React from 'react';

export const Loading = () => {
  return <p style={{fontSize: `40px`, lineHeight: `1`, textAlign: `center`}}> Loading...</p>;
};
