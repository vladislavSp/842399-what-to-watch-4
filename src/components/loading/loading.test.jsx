import React from "react";
import renderer from "react-test-renderer";
import {Loading} from "./loading";

it(`Loading renders correctly`, () => {
  const tree = renderer
    .create(<Loading />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
