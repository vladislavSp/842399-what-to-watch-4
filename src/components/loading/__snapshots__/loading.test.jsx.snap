// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<p
  style={
    Object {
      "fontSize": "40px",
      "lineHeight": "1",
      "textAlign": "center",
    }
  }
>
   Loading...
</p>
`;
